tests/openvino/test_exporters_cli.py

```diff
@@ -192,27 +192,27 @@ class OVCLIExportTestCase(unittest.TestCase):
                     "image-text-to-text",
                     "llava_next",
                     "int4 --group-size 16 --ratio 0.8",
-                    [{"int8": 14, "int4": 16}, {"int8": 9}, {"int8": 1}],
+                    [{"int8": 14, "int4": 16}, {"int8": 1}, {"int8": 9}],
```

Comment:
Order of some reference values is changed because of the addition of this line: `submodels = list(model.submodels.values())`. This alters the order of submodels compared to before.